frontend/src/app/components/Chrome/Header/BuildInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+
+class BuildInfo extends Component {
+    public state = {
+        buildDate: '',
+        commitHash: '',
+        commitText: ''
+    };
+
+    public render() {
+        this.getBuildInfo()
+        return (
+            <React.Fragment>
+                <div className="column-view">
+                    <Link to=""
+                        onClick={this.commitRedirect}>
+                        {this.state.buildDate}
+                    </Link>
+                </div>
+            </React.Fragment>
+        );
+    }
+
+    private getBuildInfo() {
+        const backend = "/pantheon/builddate.json"
+        if (this.state.buildDate === '') {
+            fetch(backend)
+                .then(response => response.json())
+                .then(responseJSON => this.setState({ buildDate: "Built Date: " + responseJSON.buildDate, commitHash: responseJSON.commitHash }))
+                .then(() => {
+                    if (!(this.state.commitHash.includes("not set"))) {
+                        this.setState({ commitText: 'commit hash' })
```

Comment:
Is this intentional? You are literally showing the words "commit hash" as the link text to the user?

Reply:
That will never be shown, it's left behind from previous commit. I will remove that now.